src/contexts/noteContext.js

```diff
@@ -139,7 +139,7 @@ export const NoteProvider = ({ children }) => {
 				note: note,
 			});
 		} catch (error) {
-			console.log("from addToNoteList", error);
+			console.log("from deleteArchivedNote", error);
 		}
```

Comment:
You can remove these console.log statements.

Reply:
Ok